### PR TITLE
Update fortran90.html.markdown

### PR DESCRIPTION
**Pull request by Ily83**
added DO CONCURRENT

- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

**Comment by nbehrnd** [comment]
Thank you for the addition of this function introduced more recently to Fortran (in Fortran's time scale).  For consistency of the format of presentation, may you please edit the comments into two-line comments, i.e., to substitute a)

``` f90
! ISO Standard Fortran 2008 introduced the DO CONCURRENT construct to allow you to express loop-level parallelism
```

by

``` f90
! ISO Standard Fortran 2008 introduced the DO CONCURRENT construct to allow you
! to express loop-level parallelism.
```

and b)

``` f90
! Only calls to pure functions are allowed inside the loop and we can declare multiple indices:
```

by

``` f90
! Only calls to pure functions are allowed inside the loop and we can declare
! multiple indices:
```

This is one of the [convened rules](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown) to ease the eventual display as [web page](https://learnxinyminutes.com/docs/fortran90/) applying a one-for-all languages style sheet (though not always working for everyone's setup, e.g. [example](https://github.com/adambard/learnxinyminutes-site/issues/31) and [example](https://github.com/adambard/learnxinyminutes-docs/issues/2611)).  Independent of the project, this restrain on the code blocks simultaneously eases to read your contributions here on GitHub's issue reports (no need for a horizontal scroll bar) and inspections of git's difference views in a terminal, too.

**Comment by Ily83** [comment]
Can I add more stuff here or should I need to do another pull request?

**Comment by nbehrnd** [comment]
Addition of @robochat (the initiator of the Fortran summary), addition of @mribeirodantas (one of those with the key to merge .and. more frequently seen).

**Comment by nbehrnd** [comment]
@Ily83 If the additional edit belongs to `do concurrent`, I suggest to amend the
current PR.  If it is independent, I would suggest a separate PR (as in atomic
changes / one additional feature described at a time).  If the edit requires
`do concurrent` to be already described, I would file a PR *after* the current
PR has been merged.

File: fortran90.html.markdown [hunk 1]
```diff
@@ -436,6 +436,36 @@ contains
 
 end module fruity
 
+
+! ISO Standard Fortran 2008 introduced the DO CONCURRENT construct to allow you
+! to express loop-level parallelis
```
Comment (vendethiel) [inline fortran90.html.markdown:441]: Typo: parallelism

Comment (nbehrnd) [reply fortran90.html.markdown:441]: @vendethiel Amended in #4878

Comment (verhovsky) [reply fortran90.html.markdown:441]: that was to the wrong branch on the main repo

**Pull request merged**